base/base/BorrowedObjectPool.h

```diff
@@ -46,7 +46,7 @@ class BorrowedObjectPool final
 
         bool has_unlimited_size = (max_size == 0);
 
-        if (unlikely(has_unlimited_size) || allocated_objects_size < max_size)
```

Comment:
I suppose `if (unlikely(has_unlimited_size) || allocated_objects_size < max_size)` is not the same as `if (has_unlimited_size || allocated_objects_size < max_size) [[unlikely]]`.

Reply:
[[likely]]/[[unlikely]] always relate to the entire if branch, whereas likely()/unlikely() may also relate to individual conditions (like here). So indeed, without rewriting (breaking up) the if construct into something much uglier, the new syntax can't be used as a 1:1 replacement for the new syntax.